apps/api/Dockerfile

```
@@ -0,0 +1,12 @@
+FROM python:3.9
+
+COPY requirements.txt requirements.txt
+
+RUN apt-get update && apt-get install -y libxml2-dev libxmlsec1-dev libxmlsec1-openssl
+RUN pip install --no-cache-dir -r requirements.txt "uvicorn[standard]"
+
+EXPOSE 8000
+
+COPY . .
```

Suggestion: We don't need to copy everything to the image, only the `src` folder.
```suggestion
COPY src/ src/
```